packages/playwright-test/src/common/compilationCache.ts

```diff
@@ -27,7 +28,11 @@ export type MemoryCache = {
 };
 
 const version = 13;
-const cacheDir = process.env.PWTEST_CACHE_DIR || path.join(os.tmpdir(), 'playwright-transform-cache');
+
+const DEFAULT_CACHE_DIR_WIN32 = path.join(os.tmpdir(), `playwright-transform-cache`);
+const DEFAULT_CACHE_DIR_POSIX = path.join(os.tmpdir(), `playwright-transform-cache-` + sanitizeForFilePath(os.userInfo().username));
```

Comment:
Hi @aslushnikov 
I encountered some issue which caused by `os.userInfo()`. u may want to use the alternative what Cypress use.
https://github.com/cypress-io/cypress/pull/17488

Reply:
Please file an GitHub issue, thanks!